Makefile.PL

```diff
@@ -31,7 +31,7 @@ requires
   'Starman'                     => 0,
   'String::ShellQuote'          => 0,
   'Try::Tiny'                   => 0.24,
-  'Zonemaster::Engine'          => 4.0,
+  'Zonemaster::Engine'          => 4.001,
```

Comment:
Does CLI make use of the new features introduced in Engine? Otherwise it's better not raise the version requirement.

Reply:
You mean Backend.

If someone updates Backend by "cpanm Zonemaster::Backend" we want that to pull updated Zonemaster-LDNS and Zonemster-Engine too.

I have followed the instruction "[Update Makefile.PL with required version]".

We have had this discussion before. We have tested CLI with the updated Engine, v4.1.0, not with v4.0.0.

[Update Makefile.PL with required version]: https://github.com/zonemaster/zonemaster/blob/develop/docs/internal-documentation/maintenance/ReleaseProcess-release.md#5-update-makefilepl-with-required-version